styles.css

```diff
@@ -1,826 +1,980 @@
+    display: none;
+  }
+
+  p.desktop-para {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    display: block;
+  }
+
+  ul.flex-icone {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list ul li {
+    list-style-type: none;
+    padding-bottom: 10px;
+  }
+
+  .flex-container-list li:first-child {
+    font-weight: 500;
+    font-size: 18px;
+    color: #344563;
+    margin-right: 32px;
+    line-height: 24px;
+  }
+
+  .flex-container-list li:nth-child(2),
+  ul li.desktop-version-li1 {
+    list-style: disc;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #7a869a;
+    margin-right: 32px;
+  }
+
+  .flex-container-list li:nth-child(2) {
+    list-style: disc;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #7a869a;
+    margin-right: 32px;
+  }
+
+  .flex-container-list li:last-child {
+    list-style: disc;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #7a869a;
+  }
+
+  .btn-see-projects a:hover {
+    background: #6070ff;
+    color: #fff;
+    box-shadow: 0 8px 16px rgb(64 83 252 / 24%);
+    border-radius: 8px;
+  }
+
+  .btn-see-projects a:active {
+    background: #2230d2;
+    border-radius: 8px;
+    color: #ffff;
+  }
+
+  .flex-lang li span {
+    padding: 4px 12px;
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    cursor: pointer;
+    letter-spacing: 0.03em;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  /* ---------Form Section------------ */
+  .container.desktop {
+    height: 577px;
+    width: 801px;
+    margin: 142px 320px 127px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contact-form {
+    width: 100%;
+    margin-top: 0;
+  }
+
+  .contact-content h1 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .contact-content p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+  }
+
+  .name-container input {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 48px;
+    padding-left: 5px;
+  }
+
+  .mail-container input {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 48px;
+    padding-left: 5px;
+  }
+
+  .textarea-container textarea {
+    font-family: Poppins, sans-serif;
+    border: 1px solid transparent;
+    border-radius: 8px;
+    background: #fff;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+    width: 80%;
+    margin-top: 20px;
+    height: 170px;
+    padding-left: 5px;
+  }
```

Comment:
- Please, kindly adjust the width of the inputs and text area as yours is still very big compared to the Figma design. Please, refer to the template on Figma to fix this issue below:

![reduce](https://user-images.githubusercontent.com/59397159/187049996-6156c332-3715-4397-8483-033f69f29af8.png)

Reply:
*optional*

- Your design is having a horizontal scroll, please fix this as this is not a good user experience on your web page